build.gradle

```diff
@@ -81,12 +81,6 @@ class ResolveIntelliJVersions extends DefaultTask {
 task resolveIntelliJVersions(type: ResolveIntelliJVersions)
 
 
-if (isCI) {
-  gradle.startParameter.configureOnDemand = false
-  gradle.startParameter.logLevel = LogLevel.INFO
-  gradle.startParameter.showStacktrace = ShowStacktrace.ALWAYS
```

Comment:
These should (hopefully) be now addressed with `testLogging` settings instead, leading to less spam from Gradle, only the failed test output that we really care about and that's not show in Gradle output by default